test/integration/example-models/education/copyright.html

```diff
@@ -1,13 +0,0 @@
-<hr>
-
-<p> Copyright (c) 2016, the author(s). All rights reserved. </p>
```

Comment:
Would we not get in trouble for removing this copyright?

Reply:
No idea, haha. I'll put it back just in case.